src/error.rs

```diff
@@ -20,6 +20,10 @@ error_chain! {
             description("unknown syntax"),
             display("unknown syntax: '{}'", name)
         }
+        InvalidPagerValueBat {
+            description("invalid value `bat` for pager property"),
+            display("the use of bat as a pager is disallow in order to avoid infinite recursion problems")
```

Comment:
```suggestion
            display("the use of bat as a pager is disallowed in order to avoid infinite recursion problems")
```

Reply:
I also think `the` should be removed from the beginning. And `use` should be capitalized. i.e. `Use`

Reply:
Fix the typo:
```
$ bat --pager="bat --style numbers" myfile.txt
[bat error]: Use of bat as a pager is disallowed in order to avoid infinite recursion problems
```